lib/data/sql/V5__export_projections.sql

```diff
@@ -192,7 +192,51 @@ COPY (
 ----- types
 
 COPY
-    (WITH additional AS
+    (SELECT jsonb_build_object(
+      'lu', (SELECT jsonb_build_object('standortTypen', (SELECT json_agg(jsonb_build_object('stoNr', sto_nr,
```

Comment:
JSON keys should be in english, similiar to other types.

Reply:
Done. Check out my English definitions and see if they are comprehensible.